setup.rst

```diff
@@ -10,57 +10,67 @@ Installing & Setting up the Symfony Framework
     Do you prefer video tutorials? Check out the `Stellar Development with Symfony`_
     screencast series.
 
-Installing Symfony
-------------------
+Technical Requirements
+----------------------
 
-Before creating your first Symfony application, make sure to meet the following
-requirements:
+Before creating your first Symfony application you must:
 
-* Your server has PHP 7.1 or higher installed (and :doc:`these PHP extensions </reference/requirements>`
+* Make sure to have PHP 7.1 or higher installed (and :doc:`these PHP extensions </reference/requirements>`
   which are installed and enabled by default by PHP);
-* You have `installed Composer`_, which is used to install PHP packages;
-* You have installed the :doc:`Symfony local web server </setup/symfony_server>`,
-  which provides all the tools you need to develop your application locally.
+* `Install Composer`_, which is used to install PHP packages;
+* `Install Symfony`_, which creates in your computer a binary called ``symfony``
+  that provides all the tools you need to develop your application locally.
 
-Once these requirements are installed, open your terminal and run any of these
-commands to create the Symfony application:
+.. _creating-symfony-applications:
+
+Creating Symfony Applications
+-----------------------------
+
+Open your console terminal and run any of these commands to create a new Symfony
+application:
 
 .. code-block:: terminal
 
     # run this if you are building a traditional web application
-    $ symfony new --full my_project
+    $ symfony new --full my_project_name
 
     # run this if you are building a microservice, console application or API
     $ symfony new my-project
 
 The only difference between these two commands is the number of packages
-installed. The ``--full`` option installs all the packages that you usually
-need to build web applications. Therefore, the installation size will be much bigger.
+installed by default. The ``--full`` option installs all the packages that you
+usually need to build web applications, so the installation size will be bigger.
 
-Both commands will create a new ``my-project/`` directory, download some
-dependencies into it and even generate the basic directories and files you'll
-need to get started. In other words, your new app is ready!
+If you can't or don't want to `install Symfony`_ for any reason, run these
+commands to create the new Symfony applications using Composer:
```

Comment:
```suggestion
commands to create the new Symfony application using Composer:
```

as upper you wrote `Open your console terminal and run any of these commands to create a new Symfony
	application:`